### ISSUE TITLE
Feel free to invite one or two people to the group

### ISSUE THREAD

**Issue by pcowgill** [decentralized-software/chicago#14]

**Comment by pcowgill** [comment]
TL;DR Feel free to share the GitHub link with others directly from now on
https://github.com/decentralized-software/chicago

--------------
Long version:

We used to encourage people to go through the Typeform to sign up to make sure they (A) read the code of conduct with our emphasis on diversity and inclusion and (B) understood our laser focus on software. But it was a little too much friction for welcoming new people to the group.

Re: (A), I added the code of conduct to the README in this PR https://github.com/decentralized-software/chicago/pull/13

As for (B), the group being run on GitHub ought to ensure that we continue to skew towards software/design/product/technical people enough to serve that purpose.

So feel free to share the group with others using the GitHub repo link going forward!

https://github.com/decentralized-software/chicago
